### PR TITLE
Fix webhook command

### PR DESCRIPTION
**Pull request by relekang**

### CODE REVIEW

File: frigg/webhooks/test_commands.py [hunk 1]
```diff
@@ -16,13 +17,25 @@ def load_fixture(self, fixture):
         fixtures_path = os.path.join(settings.BASE_DIR, self.FIXTURE_PATH, fixture)
         return json.load(open(fixtures_path, encoding='utf-8'))
 
-    def test_command(self):
-        r = redis.Redis(**settings.REDIS_SETTINGS)
-        r.lpush(settings.FRIGG_WEBHOOK_QUEUE, json.dumps({
+    def setUp(self):
+        self.maxDiff = None
```
Comment (frecar) [inline frigg/webhooks/test_commands.py]: what is this used for?

File: frigg/webhooks/test_commands.py [hunk 2]
```diff
@@ -16,13 +17,25 @@ def load_fixture(self, fixture):
...
+        self.r = redis.Redis(**settings.REDIS_SETTINGS)
```
Comment (frecar) [inline frigg/webhooks/test_commands.py]: self.r  -->  self.redis ?

File: frigg/webhooks/test_commands.py (hunk 1 above)
Comment (relekang) [inline frigg/webhooks/test_commands.py]: [Making long diff print as diffs and not something like "you have a long diff"](https://docs.python.org/3.5/library/unittest.html#unittest.TestCase.maxDiff)

File: frigg/webhooks/test_commands.py (hunk 2 above)
Comment (relekang) [inline frigg/webhooks/test_commands.py]: Fixed

**Comment by frecar** [comment]
:+1:

**Pull request merged**